service/middleware.go

```diff
@@ -254,6 +254,11 @@ func createProxyRequestMiddleware(next http.Handler, config config.Config, servi
 				for headerName, headerValue := range typedCachedResponse.HeaderMap {
 					w.Header().Add(headerName, headerValue)
 				}
+				// add CORS headers
+				accessControlAllowOriginValue := config.GetAccessControlAllowOriginValue(r.Host)
+				if accessControlAllowOriginValue != "" {
```

Comment:
important to add to the spec in the caching doc that we won't override any value of this header if present from the backend response

Reply:
actually in current implementation we override

I can add smth like this:
```go
// add CORS headers
accessControlAllowOriginValue := config.GetAccessControlAllowOriginValue(r.Host)
if w.Header().Get("Access-Control-Allow-Origin") == "" && accessControlAllowOriginValue != "" {
    w.Header().Add("Access-Control-Allow-Origin", accessControlAllowOriginValue)
}
```
@galxy25 ?